app/konnect/org-management/sso.md

```diff
@@ -61,7 +58,38 @@ You can configure custom IdP-specific behaviors in the **Advanced Settings** of
     - `name`: Used as the {{site.konnect_short_name}} account's `full_name`.
     - `email`: Used as the {{site.konnect_short_name}} account's `email`.
     - `groups`: Used to map users to teams defined in the team mappings upon login.
+{% endnavtab %}
+{% navtab SAML %}
+1. Log in to {{site.konnect_saas}}, click {% konnect_icon organizations %} **Organization**, and then select **Auth Settings**.
+
+2. Click **Configure provider** under **SAML**. 
+
+3. Enter the **Metadata URL** from your IdP in the **Identity Provider Metadata URL** field, or paste the **Metadata XML** in to the Identity **Provider Metadata XML** field.
```

Comment:
Although it is supported, we won't expose the raw XML configuration in the initial release.
```suggestion
3. Enter the **Metadata URL** from your IdP in the **IDP Metadata URL** field.
```